src/components/Gesture.svelte

```diff
@@ -270,45 +283,77 @@
           <CloseIcon class="text-xl m-1" />
         </IconButton>
       </div>
-      <h3
-        class="px-2"
-        contenteditable
-        bind:innerText={$nameBind}
-        on:click={titleClicked}
-        on:keypress={onTitleKeypress} />
-    </div>
+    {/if}
+    <label for="gestureName" class="sr-only"
+      >{$t('content.data.addAction.inputLabel')}</label>
+    <input
+      use:init
+      name="gestureName"
+      class="w-40 col-start-2 p-2 col-end-5 transition ease rounded bg-gray-100 placeholder-gray-500 outline-primary"
+      contenteditable
```

Comment:
We don't need `contenteditable` now you've made this a proper input.

Reply:
https://github.com/microbit-foundation/ml-trainer/pull/103/commits/acb2ba8030fe844abacc5c2eb4218048b18713fa